docs/CONTRIBUTING.md

```diff
@@ -78,12 +78,12 @@ we'll guide you.
 
 ## Development workflow
 
-The current development workflow is a bit tricky because we have this repo and we use our `cdr/vscode` fork inside it with [`yarn link`](https://classic.yarnpkg.com/lang/en/docs/cli/link/).
+The current development workflow is a bit tricky because we have this repo and we use our `coderer/vscode` fork inside it with [`yarn link`](https://classic.yarnpkg.com/lang/en/docs/cli/link/).
```

Comment:
```suggestion
The current development workflow is a bit tricky because we have this repo and we use our `coder/vscode` fork inside it with [`yarn link`](https://classic.yarnpkg.com/lang/en/docs/cli/link/).
```